packages/cascara/src/ui/Form/Form.js

```diff
@@ -170,11 +176,18 @@ const Form = ({
       ? Boolean(actions?.modules?.find((action) => action.module === 'edit'))
       : incomingIsEditable;
 
-  const [isEditing, setIsEtiding] = useState(() =>
-    !isEditable ? false : isInitialEditing
-  );
+  const [isEditing, setIsEtiding] = useState(() => {
+    const isDataEmpty = isEmpty(data);
+    if (isDataEmpty) {
+      return true;
+    }
+    return !isEditable ? false : isInitialEditing;
+  });
```

Comment:
Let's explore setting a message with useDeveloperMessage if `isLoading && !isInitialEditing` as this is an invalid state.

Reply:
File global for Form is added, just like it is in Pagination.